assets/model_monitoring/components/src/action_analyzer/contracts/index_action_contracts.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Action sample class."""
+
+import datetime
+import uuid
+import json
+from enum import Enum
+from abc import ABC, abstractmethod
+from action_analyzer.contracts.action import ActionSample
+from action_analyzer.utils.utils import convert_to_camel_case
+
+
+class IndexActionSample(ActionSample):
+    """Index Action sample class."""
+
+    def __init__(self,
+                 question: str,
```

Comment:
what is the question and answer in this type of action sample? and can different type of action samples have different meanings of question and answer?